scr/sysbox

```diff
@@ -58,12 +58,19 @@ function load_required_modules() {
 
 # Returns linux distro running in the system.
 function get_host_distro() {
-
   local distro=$(cat /etc/os-release | awk -F"=" '/^ID=/ {print $2}' | tr -d '"')
-
   echo $distro
 }
 
+# Ensures user "sysbox" is present on the host
```

Comment:
Good idea to make the user a no-login user. I added that, and in addition made user "sysbox" a system user (so that it get uid:gid in the system's range rather than in the user range). 

Regarding the installer, I think the it should probably follow the way we do it in the test script, but I don't feel too comfortable making that change since I don't know the installer code too well or how to test it.